controllers/perconaservermysql_controller.go

```diff
@@ -356,8 +381,7 @@ func (r *PerconaServerMySQLReconciler) reconcileVersions(ctx context.Context, cr
 
 	err = r.Patch(ctx, cr.DeepCopy(), patch)
 	if err != nil {
-		log.Info("Failed to update CR, using the default version", "error", err)
-		return nil
+		return errors.Wrap(err, "failed to update CR, using the default version")
```

Comment:
Same as for `failed to get versions, using the default ones` case.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/291/commits/cf68585fb494946f31a373103334d5c0c0a1a51f